README.md

```diff
@@ -37,9 +37,16 @@ Add this line to your application's deploy.rb file:
 The below example ensures that there are 3 users existing in the database after running the 'rake reference_data:load'
 
     ### db/reference/000_users.rb
-    Easy::ReferenceData.refresh User, :system_code, 'nigel', name: 'Nigel Ramsay', email: 'nigel.ramsay@mailinator.com'
-    Easy::ReferenceData.refresh User, :system_code, 'fred', name: 'Fred Schmitt', email: 'fred.schmitt@mailinator.com'
-    Easy::ReferenceData.refresh User, :system_code, 'bert', name: 'Bert Symthe', email: 'bert.smythe@mailinator.com'
+    Easy::ReferenceData.update_or_create User, system_code: 'nigel', name: 'Nigel Ramsay', email: 'nigel.ramsay@mailinator.com', keys: [:system_code]
```

Comment:
Should these examples be adjusted to separate the `keys` option from the `User` attributes? Eg:

```
Easy::ReferenceData.update_or_create User, {system_code: 'nigel', name: 'Nigel Ramsay', email: 'nigel.ramsay@mailinator.com'}, keys: [:system_code]
```